packages/matter-node.js/test/IntegrationTest.ts

```diff
@@ -116,6 +117,23 @@ describe("Integration Test", () => {
         onOffLightDeviceServer = new OnOffLightDevice();
         commissioningServer.addDevice(onOffLightDeviceServer);
 
+        // Override NetworkCommissioning Cluster for now unless configurable
+        commissioningServer.addRootClusterServer(
+            ClusterServer(
+                WifiAndEthernetAndThreadNetworkCommissioningCluster,
+                {
+                    maxNetworks: 1,
+                    interfaceEnabled: true,
+                    lastConnectErrorValue: 0,
+                    lastNetworkId: ByteArray.fromHex("0000000000000000000000000000000000000000000000000000000000000000"),
```

Comment:
I think you can use new ByteArray(length) to get a zero-initialized ByteArray

Reply:
Yes but this is also that way in oher places. Need to clean that up soon anywaay, so decided to stay that way.